static/js/views/discovered-addon.js

```diff
@@ -123,6 +128,52 @@ class DiscoveredAddon {
         controlDiv.innerHTML = el;
       });
   }
+
+  /**
+   * Handle a click on the license button.
+   */
+  handleLicense(e) {
+    if (e.target.dataset.id) {
+      API.getAddonsInfo().then((data) => {
+        const licenseUrl = `${data.urls[0]}/license/${e.target.dataset.id}`;
+        let modal = document.getElementById('media-modal');
+        if (modal == null) {
+          const modalContainer = document.createElement('div');
+          modalContainer.className = 'media-modal';
+          modalContainer.id = 'media-modal';
+          const modalFrame = document.createElement('div');
+          modalFrame.className = 'media-modal-frame';
+          const content = `'<div class="media-modal-close" 
```

Comment:
There is a stray `'` inside the template string, before the `<div>` tag.

Reply:
well spotted